# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1251308701 14400
#      Wed Aug 26 13:45:01 2009 -0400
# Node ID 86787a56071271df504bf33201b301bbd0e5cf8d
# Parent  d3fccc4c4b9ea4876bd96f34d615a021670dc087
octave-bug.in: correct substitutions

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-08-26  John W. Eaton  <jwe@octave.org>
+
+	* octave-bug.in: Correct substitutions.
+
 2009-08-26  John W. Eaton  <jwe@octave.org>
 
 	* aclocal.m4 (OCTAVE_CHECK_LIBRARY): Allow empty list of header files.
 	* configure.in: Use OCTAVE_CHECK_LIBRARY to check for arpack.
 	* octave-bug.in, octave-bug.cc.in: Substitute ARPACK_CPPFLAGS and
 	ARPACK_LDFLAGS.
 	* Makeconf.in (ARPACK_CPPFLAGS, ARPACK_LDFLAGS): New variables.
 	(do-subst-config-vals): Substitute them.
diff --git a/octave-bug.in b/octave-bug.in
--- a/octave-bug.in
+++ b/octave-bug.in
@@ -48,72 +48,72 @@ CXXPICFLAG=%OCTAVE_CONF_CXXPICFLAG%
 LD_CXX=%OCTAVE_CONF_LD_CXX%
 LDFLAGS=%OCTAVE_CONF_LDFLAGS%
 LIBFLAGS=%OCTAVE_CONF_LIBFLAGS%
 RLD_FLAG=%OCTAVE_CONF_RLD_FLAG%
 LEXLIB=%OCTAVE_CONF_LEXLIB%
 LIBGLOB=%OCTAVE_CONF_LIBGLOB%
 LIBS=%OCTAVE_CONF_LIBS%
 
-AMD_CPPFLAGS=%AMD_CPPFLAGS%
-AMD_LDFLAGS=%AMD_LDFLAGS%
-AMD_LIBS=%AMD_LIBS%
-ARPACK_CPPFLAGS=%ARPACK_CPPFLAGS%
-ARPACK_LDFLAGS=%ARPACK_LDFLAGS%
-ARPACK_LIBS=%ARPACK_LIBS%
-BLAS_LIBS=%BLAS_LIBS%
-CAMD_CPPFLAGS=%CAMD_CPPFLAGS%
-CAMD_LDFLAGS=%CAMD_LDFLAGS%
-CAMD_LIBS=%CAMD_LIBS%
-CARBON_LIBS=%CARBON_LIBS%
-CCOLAMD_CPPFLAGS=%CCOLAMD_CPPFLAGS%
-CCOLAMD_LDFLAGS=%CCOLAMD_LDFLAGS%
-CCOLAMD_LIBS=%CCOLAMD_LIBS%
-CHOLMOD_CPPFLAGS=%CHOLMOD_CPPFLAGS%
-CHOLMOD_LDFLAGS=%CHOLMOD_LDFLAGS%
-CHOLMOD_LIBS=%CHOLMOD_LIBS%
-COLAMD_CPPFLAGS=%COLAMD_CPPFLAGS%
-COLAMD_LDFLAGS=%COLAMD_LDFLAGS%
-COLAMD_LIBS=%COLAMD_LIBS%
-CURL_CPPFLAGS=%CURL_CPPFLAGS%
-CURL_LDFLAGS=%CURL_LDFLAGS%
-CURL_LIBS=%CURL_LIBS%
-CXSPARSE_CPPFLAGS=%CXSPARSE_CPPFLAGS%
-CXSPARSE_LDFLAGS=%CXSPARSE_LDFLAGS%
-CXSPARSE_LIBS=%CXSPARSE_LIBS%
-DL_LIBS=%DL_LIBS%
-FFTW3_CPPFLAGS=%FFTW3_CPPFLAGS%
-FFTW3_LDFLAGS=%FFTW3_LDFLAGS%
-FFTW3_LIBS=%FFTW3_LIBS%
-FFTW3F_CPPFLAGS=%FFTW3F_CPPFLAGS%
-FFTW3F_LDFLAGS=%FFTW3F_LDFLAGS%
-FFTW3F_LIBS=%FFTW3F_LIBS%
-GRAPHICS_LIBS=%GRAPHICS_LIBS%
-GLPK_CPPFLAGS=%GLPK_CPPFLAGS%
-GLPK_LDFLAGS=%GLPK_LDFLAGS%
-GLPK_LIBS=%GLPK_LIBS%
-HDF5_CPPFLAGS=%HDF5_CPPFLAGS%
-HDF5_LDFLAGS=%HDF5_LDFLAGS%
-HDF5_LIBS=%HDF5_LIBS%
-OPENGL_LIBS=%OPENGL_LIBS%
-PTHREAD_CFLAGS=%PTHREAD_CFLAGS%
-PTHREAD_LIBS=%PTHREAD_LIBS%
-QHULL_CPPFLAGS=%QHULL_CPPFLAGS%
-QHULL_LDFLAGS=%QHULL_LDFLAGS%
-QHULL_LIBS=%QHULL_LIBS%
-QRUPDATE_LIBS=%QRUPDATE_LIBS%
-READLINE_LIBS=%READLINE_LIBS%
-REGEX_LIBS=%REGEX_LIBS%
-TERM_LIBS=%TERM_LIBS%
-UMFPACK_LIBS=%UMFPACK_LIBS%
-X11_INCFLAGS=%X11_INCFLAGS%
-X11_LIBS=%X11_LIBS%
-Z_CPPFLAGS=%Z_CPPFLAGS%
-Z_LDFLAGS=%Z_LDFLAGS%
-Z_LIBS=%Z_LIBS%
+AMD_CPPFLAGS=%OCTAVE_CONF_AMD_CPPFLAGS%
+AMD_LDFLAGS=%OCTAVE_CONF_AMD_LDFLAGS%
+AMD_LIBS=%OCTAVE_CONF_AMD_LIBS%
+ARPACK_CPPFLAGS=%OCTAVE_CONF_ARPACK_CPPFLAGS%
+ARPACK_LDFLAGS=%OCTAVE_CONF_ARPACK_LDFLAGS%
+ARPACK_LIBS=%OCTAVE_CONF_ARPACK_LIBS%
+BLAS_LIBS=%OCTAVE_CONF_BLAS_LIBS%
+CAMD_CPPFLAGS=%OCTAVE_CONF_CAMD_CPPFLAGS%
+CAMD_LDFLAGS=%OCTAVE_CONF_CAMD_LDFLAGS%
+CAMD_LIBS=%OCTAVE_CONF_CAMD_LIBS%
+CARBON_LIBS=%OCTAVE_CONF_CARBON_LIBS%
+CCOLAMD_CPPFLAGS=%OCTAVE_CONF_CCOLAMD_CPPFLAGS%
+CCOLAMD_LDFLAGS=%OCTAVE_CONF_CCOLAMD_LDFLAGS%
+CCOLAMD_LIBS=%OCTAVE_CONF_CCOLAMD_LIBS%
+CHOLMOD_CPPFLAGS=%OCTAVE_CONF_CHOLMOD_CPPFLAGS%
+CHOLMOD_LDFLAGS=%OCTAVE_CONF_CHOLMOD_LDFLAGS%
+CHOLMOD_LIBS=%OCTAVE_CONF_CHOLMOD_LIBS%
+COLAMD_CPPFLAGS=%OCTAVE_CONF_COLAMD_CPPFLAGS%
+COLAMD_LDFLAGS=%OCTAVE_CONF_COLAMD_LDFLAGS%
+COLAMD_LIBS=%OCTAVE_CONF_COLAMD_LIBS%
+CURL_CPPFLAGS=%OCTAVE_CONF_CURL_CPPFLAGS%
+CURL_LDFLAGS=%OCTAVE_CONF_CURL_LDFLAGS%
+CURL_LIBS=%OCTAVE_CONF_CURL_LIBS%
+CXSPARSE_CPPFLAGS=%OCTAVE_CONF_CXSPARSE_CPPFLAGS%
+CXSPARSE_LDFLAGS=%OCTAVE_CONF_CXSPARSE_LDFLAGS%
+CXSPARSE_LIBS=%OCTAVE_CONF_CXSPARSE_LIBS%
+DL_LIBS=%OCTAVE_CONF_DL_LIBS%
+FFTW3_CPPFLAGS=%OCTAVE_CONF_FFTW3_CPPFLAGS%
+FFTW3_LDFLAGS=%OCTAVE_CONF_FFTW3_LDFLAGS%
+FFTW3_LIBS=%OCTAVE_CONF_FFTW3_LIBS%
+FFTW3F_CPPFLAGS=%OCTAVE_CONF_FFTW3F_CPPFLAGS%
+FFTW3F_LDFLAGS=%OCTAVE_CONF_FFTW3F_LDFLAGS%
+FFTW3F_LIBS=%OCTAVE_CONF_FFTW3F_LIBS%
+GRAPHICS_LIBS=%OCTAVE_CONF_GRAPHICS_LIBS%
+GLPK_CPPFLAGS=%OCTAVE_CONF_GLPK_CPPFLAGS%
+GLPK_LDFLAGS=%OCTAVE_CONF_GLPK_LDFLAGS%
+GLPK_LIBS=%OCTAVE_CONF_GLPK_LIBS%
+HDF5_CPPFLAGS=%OCTAVE_CONF_HDF5_CPPFLAGS%
+HDF5_LDFLAGS=%OCTAVE_CONF_HDF5_LDFLAGS%
+HDF5_LIBS=%OCTAVE_CONF_HDF5_LIBS%
+OPENGL_LIBS=%OCTAVE_CONF_OPENGL_LIBS%
+PTHREAD_CFLAGS=%OCTAVE_CONF_PTHREAD_CFLAGS%
+PTHREAD_LIBS=%OCTAVE_CONF_PTHREAD_LIBS%
+QHULL_CPPFLAGS=%OCTAVE_CONF_QHULL_CPPFLAGS%
+QHULL_LDFLAGS=%OCTAVE_CONF_QHULL_LDFLAGS%
+QHULL_LIBS=%OCTAVE_CONF_QHULL_LIBS%
+QRUPDATE_LIBS=%OCTAVE_CONF_QRUPDATE_LIBS%
+READLINE_LIBS=%OCTAVE_CONF_READLINE_LIBS%
+REGEX_LIBS=%OCTAVE_CONF_REGEX_LIBS%
+TERM_LIBS=%OCTAVE_CONF_TERM_LIBS%
+UMFPACK_LIBS=%OCTAVE_CONF_UMFPACK_LIBS%
+X11_INCFLAGS=%OCTAVE_CONF_X11_INCFLAGS%
+X11_LIBS=%OCTAVE_CONF_X11_LIBS%
+Z_CPPFLAGS=%OCTAVE_CONF_Z_CPPFLAGS%
+Z_LDFLAGS=%OCTAVE_CONF_Z_LDFLAGS%
+Z_LIBS=%OCTAVE_CONF_Z_LIBS%
 
 DEFS=%OCTAVE_CONF_DEFS%
 
 : ${USER=$LOGNAME}
 
 CC_AND_VERSION=
 if test -n "$CC_VERSION"; then
   CC_AND_VERSION="$CC, version $CC_VERSION"
